src/Meshes/meshLODLevel.ts

```diff
@@ -8,12 +8,12 @@ import { Nullable } from '../types';
 export class MeshLODLevel {
     /**
      * Creates a new LOD level
-     * @param distance defines the distance where this level should star being displayed
+     * @param distanceOrScreenCoverage defines either the distance or the screen coverage where this level should start being displayed
      * @param mesh defines the mesh to use to render this level
      */
     constructor(
-        /** Defines the distance where this level should start being displayed */
-        public distance: number,
+        /** Either distance from the center of the object to show this level or the screen coverage if `useLODScreenCoverage` is set to `true` on the mesh*/
+        public distanceOrScreenCoverage: number,

```

Comment:
You can't really rename the property because it would be a breaking change...

Or maybe we can accept it (@deltakosh / @sebavan ?) as I'm not sure anyone should directly work with the `MeshLODLevel` class, in which case the **Breaking changes** section of the what's new file should be updated.

Reply:
It is ok as this is not binding for either TS or JS :)